### PR TITLE
QCEngine

### PR DESCRIPTION
**Pull request by dgasmith**
A compute engine for quantum chemistry.

### CODE REVIEW

File: recipes/qcengine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+about:
+  home: http://github.com/MolSSI/QCEngine
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A wrapper for Quantum Chemistry engines that adheres to the MolSSI QCSchema.'
+
+#  description: |
+#  doc_url: http://simplejson.readthedocs.io/
```
Comment (jakirkham) [inline recipes/qcengine/meta.yaml]: Please drop these.

File: recipes/qcengine/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+  run:
+    - python
+    - pyyaml
+    - py-cpuinfo
+    - psutil
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
```
Comment (jakirkham) [inline recipes/qcengine/meta.yaml]: Also can drop this too.

**Comment by jakirkham** [comment]
Couple minor comments. Otherwise LGTM.

**Comment by dgasmith** [comment]
Thanks! Comments dropped or updated.

**Comment by marcelotrevisani** [comment]
I'm going to merge this! LGTM!

Thanks to put this in ``conda-forge``!  :)

**Pull request merged**